### PR TITLE
Remove oc tool wrapper #593

### PR DESCRIPTION
**Pull request by andrejonathan**

### CODE REVIEW

File: src/gluon/events/team/MemberRemovedFromTeam.ts [hunk 1]
```diff
@@ -120,12 +118,9 @@ export class MemberRemovedFromTeam extends BaseQMEvent implements HandleEvent<an
                 }
             }
         } catch (error) {
-            if (error instanceof OCCommandResult) {
-                throw new OCResultError(
+                throw new QMError(
```
Comment (kbristow) [inline src/gluon/events/team/MemberRemovedFromTeam.ts]: What errors can be thrown here? Why were we specifically checking for instance of OCCommandResult errors?

File: src/gluon/events/team/MembersAddedToTeam.ts [hunk 2]
```diff
@@ -150,10 +148,7 @@ export class MembersAddedToTeam extends BaseQMEvent implements HandleEvent<any>
                 }
             }
         } catch (error) {
-            if (error instanceof OCCommandResult) {
-                throw new OCResultError(error, `Failed to add openshift team member permissions to the team projects.`);
-            }
-            throw error;
+            throw new QMError(error, `Failed to add OpenShift team member permissions to the team projects.`);
```
Comment (kbristow) [inline src/gluon/events/team/MembersAddedToTeam.ts]: Same thing here.

File: src/gluon/events/team/MemberRemovedFromTeam.ts (hunk 1 above)
Comment (andrejonathan) [reply src/gluon/events/team/MemberRemovedFromTeam.ts]: Removed as discussed

File: src/gluon/events/team/MembersAddedToTeam.ts (hunk 2 above)
Comment (andrejonathan) [reply src/gluon/events/team/MembersAddedToTeam.ts]: Removed as discussed

**Comment by andrejonathan** [comment]
Added https://github.com/absa-subatomic/quantum-mechanic/issues/628 for QMError codes

**Pull request merged**